metadata.rb

```diff
@@ -6,5 +6,5 @@
 long_description 'Installs/Configures deploy-user'
 version '1.4.2'
 
-depends 'ssh_known_hosts', '~> 0.2'
+depends 'ssh_known_hosts', '~> 4.0'
```

Comment:
the ssh_known_hosts is not the one from the supermarket. There is no 4.0 for https://github.com/kierenevans/chef-ssh-known-hosts/releases